Analysis/Tasks/PWGHF/HFLcCandidateSelector.cxx

```diff
@@ -281,7 +282,7 @@ struct HFLcCandidateSelector {
   {
     int statusLcpKpi, statusLcpiKp; // final selection flag : 0-rejected  1-accepted
     bool topolLcpKpi, topolLcpiKp;
-    int pidLc, proton, kaonMinus, pionPlus;
+    int pidLcpKpi, pidLcpiKp, proton1, proton2, kaonMinus, pionPlus1, pionPlus2;
```

Comment:
Most of these variables don't need to be declared here and initialised a bit later. Instead they can be declared and initialised directly with the results of the related functions.